views/includes/footer.hbs

```diff
@@ -28,3 +28,4 @@
   </a>
   </ul>
 <footer>
+<script async src="https://www.google-analytics.com/analytics.js"></script>
```

Comment:
I had the same thoughts, but found that most examples available include the analytics script directly on the HTML, so decided to go with this solution for now. We may experiment with moving it to analytics.js in the future.